book/src/configuration.md

```diff
@@ -171,6 +171,7 @@ All git related options are only enabled in a git repository.
 |`deduplicate-links` | Ignore symlinks that point at files already shown in the picker | true
 |`parents` | Enables reading ignore files from parent directories | true
 |`ignore` | Enables reading `.ignore` files | true
+|`global-ignore` | Enables reading a global `.ignore` file from the same directory as the config.toml configuration | true
```

Comment:
Why do we need a feature flag? If the user doesn't have an ignore file then this feature is effectively disabled anyway

Reply:
It's mentioned in the OP.
> I considered just checking whether the file exists and adding it [...]

Should I switch to that approach?
Where should I put the info about the global .ignore? configuration.md or some place else?

Reply:
How's the current version?